pymc_experimental/tests/test_linearmodel.py

```diff
@@ -101,6 +112,15 @@ def test_fit(fitted_linear_model_instance):
     assert isinstance(post_pred, xr.DataArray)
 
 
+def test_parameter_fit(fitted_linear_model_instance, toy_actual_params):
```

Comment:
Nitpick, we should actually fit in the test if we want to check it worked correctly.

The `fitted_model` used as a fixture should have a dummy fit that is fast enough, for tests that are not really testing the quality of fit.

Reply:
That's a good thought. I can work on that.